prow/plugins.yaml

```diff
@@ -2,50 +2,63 @@
 # Keys: Full repo name: "org/repo".
 # Values: List of plugins to run against the repo.
 ---
-google/cadvisor:
```

Comment:
Not sure what you mean -- the two changes are:

1. take entirety of current `plugins.yaml`, indent it once and call it `plugins:`
2. copy-paste plugin config from `config.yaml` into `plugins.yaml` at the top level